packages/ndla-image-search/src/PreviewImage.js

```diff
@@ -23,11 +23,10 @@ export default function PreviewImage({ image, onSelectImage, useImageTitle }) {
   return (
     <div {...classes()}>
       <div {...classes('image')}>
-        <img
-          alt="presentation"
-          role="presentation"
-          srcSet={getSrcSets(encodeURI(image.imageUrl))}
-        />
+        <picture>
+          <source srcSet={getSrcSets(encodeURI(image.imageUrl))} sizes="15vw" />
```

Comment:
Kan ikke default til 15vw her.  Da vil for lite på mobil. Sansynligvis noe sånt `(min-width: 800px) 15vw,100vw), men må testes.

Reply:
!5 blir vel også litt lite på store skjermer. Bør vel være 50vw?

Reply:
Har nok misforstått litt hvordan sizes kan brukes.
Skal gjøre et nytt forsøk, og teste mer, mot mobil/responsiv.